outsource/outsource.py

```diff
@@ -338,6 +375,10 @@ def _generate_workflow(self):
         combinepy = File("combine.py")
         rc.add_replica("local", "combine.py", f"file://{base_dir}/workflow/combine.py")
 
+        # script to install packages
+        combinepy = File("install.sh")
```

Comment:
I think you want to say `installpy` instead of `combinepy`

Reply:
Sure, yes.